.github/workflows/android.yaml

```diff
@@ -46,3 +50,55 @@ jobs:
     with:
       platform: android-arm
       nightly: ${{ github.event.inputs.nightly }}
+      keep_artifacts: cobalt.apk
+
+  final_android_job:
```

Comment:
As mentioned in a comment below I think this needs an `if: ${{ github.event.release.name }}` or something along those lines.

Reply:
The intent is to run this on every PR, not gated on releases. For releases, extra code needs to be added at a later stage to post the results in Releases API.